### PR TITLE
HBASE-24024 : Reject multi() requests with rows higher than threshold

### PR DESCRIPTION
**Pull request by virajjasani**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 17s |  Docker failed to build yetus/hbase:6d4e15a7ac.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/1/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  5s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 50s |  master passed  |
| +1 :green_heart: |  checkstyle  |   2m 20s |  master passed  |
| +0 :ok: |  refguide  |   5m 18s |  branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  spotbugs  |  12m  1s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 35s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   2m 18s |  root: The patch generated 0 new + 64 unchanged - 4 fixed = 64 total (was 68)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +0 :ok: |  refguide  |   5m 21s |  patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  hadoopcheck  |   6m  6s |  Patch does not cause any errors with Hadoop 3.1.2.  |
| +1 :green_heart: |  spotbugs  |  12m 38s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 34s |  The patch does not generate ASF License warnings.  |
|  |   |  63m 59s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Optional Tests | dupname asflicense refguide xml spotbugs hadoopcheck hbaseanti checkstyle |
| uname | Linux 20444fc1e640 4.15.0-91-generic #92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 5c53b606a9 |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/1/artifact/yetus-general-check/output/branch-site/book.html |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/1/artifact/yetus-general-check/output/patch-site/book.html |
| Max. process+thread count | 122 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  6s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 31s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 51s |  master passed  |
| +1 :green_heart: |  compile  |   2m 29s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 19s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m  6s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 39s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 27s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 27s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 23s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 336m 19s |  root in the patch passed.  |
|  |   | 369m 43s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux eae272a00e39 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 5c53b606a9 |
| Default Java | 1.8.0_232 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/1/testReport/ |
| Max. process+thread count | 5068 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 12s |  Docker failed to build yetus/hbase:6d4e15a7ac.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/2/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  6s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 21s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 38s |  master passed  |
| +1 :green_heart: |  checkstyle  |   2m 19s |  master passed  |
| +0 :ok: |  refguide  |   5m 21s |  branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  spotbugs  |  12m  2s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 34s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   2m 20s |  root: The patch generated 0 new + 64 unchanged - 4 fixed = 64 total (was 68)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
| +0 :ok: |  refguide  |   5m 24s |  patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  hadoopcheck  |   5m 59s |  Patch does not cause any errors with Hadoop 3.1.2.  |
| +1 :green_heart: |  spotbugs  |  12m 36s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 34s |  The patch does not generate ASF License warnings.  |
|  |   |  63m 41s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Optional Tests | dupname asflicense refguide xml spotbugs hadoopcheck hbaseanti checkstyle |
| uname | Linux a75b57278bce 4.15.0-91-generic #92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 5c53b606a9 |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/2/artifact/yetus-general-check/output/branch-site/book.html |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/2/artifact/yetus-general-check/output/patch-site/book.html |
| Max. process+thread count | 122 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  7s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  2s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 20s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 36s |  master passed  |
| +1 :green_heart: |  compile  |   2m 26s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 17s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m  1s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 36s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 26s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 26s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 21s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m  0s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 353m 49s |  root in the patch passed.  |
|  |   | 386m 29s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/2/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 095b88a35279 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 5c53b606a9 |
| Default Java | 1.8.0_232 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/2/testReport/ |
| Max. process+thread count | 5020 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-common/src/main/resources/hbase-default.xml [hunk 1]
```diff
@@ -1946,4 +1946,15 @@ possible configurations would overwhelm and obscure the important.
       responses with complete data.
     </description>
   </property>
+  <property>
+    <name>hbase.rpc.rows.size.threshold.reject</name>
+    <value>false</value>
+    <description>
+      If value is true, RegionServer will abort RPC multi() requests with number of rows
```
Comment (Reidddddd) [inline hbase-common/src/main/resources/hbase-default.xml:1953]: Some improvements is needed here.
Client/Admin, if he/she is not a developer or like else, doesn't know what is multi() request. So, here we could be more clear and informative. (In fact, it's just a batch of Puts/Deletes)

**Comment by Reidddddd** [comment]
LGTM, just some documentation stuff.

File: hbase-common/src/main/resources/hbase-default.xml (hunk 1 above)
Comment (virajjasani) [reply hbase-common/src/main/resources/hbase-default.xml:1953]: Oh, yes, I didn't realize this. Sure, let me update.
Thanks @Reidddddd

Comment (virajjasani) [reply hbase-common/src/main/resources/hbase-default.xml:1953]: Updating this to:
```
If value is true, RegionServer will abort batch requests of Put/Delete with number of rows
in a batch operation exceeding.......
```

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 36s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 36s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 33s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 56s |  master passed  |
| +0 :ok: |  refguide  |   4m 45s |  branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  spotbugs  |  12m 25s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 35s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   2m 16s |  root: The patch generated 0 new + 64 unchanged - 4 fixed = 64 total (was 68)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +0 :ok: |  refguide  |   5m 55s |  patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  hadoopcheck  |  13m  7s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1.  |
| +1 :green_heart: |  spotbugs  |  14m 27s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 39s |  The patch does not generate ASF License warnings.  |
|  |   |  72m 52s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Optional Tests | dupname asflicense refguide xml spotbugs hadoopcheck hbaseanti checkstyle |
| uname | Linux c7e4e7f7a8bf 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bc9184ee00 |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-general-check/output/branch-site/book.html |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-general-check/output/patch-site/book.html |
| Max. process+thread count | 137 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 42s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 43s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 47s |  master passed  |
| +1 :green_heart: |  compile  |   2m 57s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 52s |  branch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 23s |  hbase-common in master failed.  |
| -0 :warning: |  javadoc  |   0m 17s |  root in master failed.  |
| -0 :warning: |  javadoc  |   0m 47s |  hbase-server in master failed.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 34s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m  5s |  the patch passed  |
| +1 :green_heart: |  javac  |   3m  5s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m 39s |  patch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 16s |  hbase-common in the patch failed.  |
| -0 :warning: |  javadoc  |   0m 44s |  hbase-server in the patch failed.  |
| -0 :warning: |  javadoc  |   0m 18s |  root in the patch failed.  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 230m 38s |  root in the patch passed.  |
|  |   | 267m 18s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 8fd3590198e1 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bc9184ee00 |
| Default Java | 2020-01-14 |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-common.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-root.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-server.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-common.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-server.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/testReport/ |
| Max. process+thread count | 4961 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 36s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 45s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 36s |  master passed  |
| +1 :green_heart: |  compile  |   2m 17s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 58s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 10s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 38s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 38s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 42s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 58s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 291m  7s |  root in the patch failed.  |
|  |   | 324m 27s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1560 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux bef8c584d4d7 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bc9184ee00 |
| Default Java | 1.8.0_232 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/artifact/yetus-jdk8-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/testReport/ |
| Max. process+thread count | 5658 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1560/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**